assets/js/components/KeyMetrics/constants.js

```diff
@@ -30,23 +30,32 @@ export const MIN_SELECTED_METRICS_COUNT = 2;
 export const MAX_SELECTED_METRICS_COUNT = 4;
 export const MAX_SELECTED_METRICS_COUNT_WITH_CONVERSION_EVENTS = 8;
 
+export const KEY_METRICS_CURRENT_SELECTION_GROUP_SLUG = 'current-selection';
 export const KEY_METRICS_GROUP_VISITORS = {
 	SLUG: 'visitors',
 	LABEL: __( 'Visitors', 'google-site-kit' ),
 };
 export const KEY_METRICS_GROUP_DRIVING_TRAFFIC = {
-	SLUG: 'drivingTraffic',
+	SLUG: 'driving-traffic',
```

Comment:
I updated group slugs for consistency, since I noticed all other constants were using this format